lambdas/service/app.py

```diff
@@ -162,6 +165,13 @@ def health(keys: Optional[str] = None):
         return {
             'up': True,
         }
+    elif keys == 'cached':
+        health_object = json.loads(storage_service.get('health/service'))
+        max_age = 2 * 60
+        if time.time() - health_object['time'] > max_age:
+            return Response(body=json.dumps({'Message': 'Cached health object is stale'}), status_code=500)
```

Comment:
```suggestion
            return Response(status_code=500, 
                            body=json.dumps({'Message': 'Cached health object is stale'}))
```

Just looks better, IMO